src/types.ts

```diff
@@ -7,6 +7,7 @@ export interface CodegenOptionsSwift extends JavaLikeOptions {
 	relativeSourceOutputPath: string
 	customTemplatesPath: string | null
 	hideGenerationTimestamp: boolean
+	customRetryStatusCodes: string[]
```

Comment:
Given the implementation I think we should rename this to `additionalRetryStatusCodes` or perhaps we leave it with this name and change it so it defaults to just 429.

Could we add a quick jsdoc here:

```javascript
/** Additional HTTP response status codes that will trigger an automatic retry. 429 Too Many Requests is always automatically retried. */
```